org.gtkhash.gtkhash.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: org.gtkhash.gtkhash
```

Comment:
The `gtkhash.org` doesn't seem to exists. You should use `com.github.tristanheaven.gtkhash` or another domain name you own.

Reply:
It's registered by me, just doesn't have any A records.